### PR TITLE
Fix IR details - use import instead of require

### PR DESCRIPTION
**Pull request by jgoley**
[PT Bug](https://www.pivotaltracker.com/story/show/142947695)

### CODE REVIEW

File: imports/ui/helpers.coffee [hunk 1]
```diff
@@ -1,6 +1,5 @@
 formatLocation = require '/imports/formatLocation.coffee'
```
Comment (nathanathan) [inline imports/ui/helpers.coffee]: Can we switch this require call as well?

Comment (jgoley) [inline imports/ui/helpers.coffee]: Yep. I've made this change. I wonder if we should work towards a consistent standard—maybe update to the ES6 pattern as we make changes?

Comment (nathanathan) [inline imports/ui/helpers.coffee]: That sounds good to me

**Comment by nathanathan** [comment]
👍

**Pull request merged**